integration/test/chartvalues/chartvalues_test.go

```diff
@@ -0,0 +1,44 @@
+// +build k8srequired
+
+package chartvalues
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/giantswarm/micrologger"
+
+	"github.com/giantswarm/chart-operator/integration/chart"
+	"github.com/giantswarm/chart-operator/integration/templates"
+)
+
+func TestChartValues(t *testing.T) {
+	const cr = "chart-operator-resource"
+
+	charts := []chart.Chart{
+		{
+			Channel: "1-0-beta",
+			Release: "1.0.0",
+			Tarball: "/e2e/fixtures/tb-chart-1.0.0.tgz",
+			Name:    "tb-chart",
+		},
+	}
+
+	// Setup
+	l, err := micrologger.New(micrologger.Config{})
+	if err != nil {
+		t.Fatalf("could not create logger %v", err)
+	}
```

Comment:
The logger is already initialized and we should just reuse it here. Lets make sure that our tests are configurable just like we inject all necessary settings and dependencies in our service implementations.